src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const typeCastFunc = (item) => {
+  const element = item.replace(/\$/g, '');
```

Comment:
```suggestion
  const element = item.replace(/[$,]/g, '');
```
це мало б одразу і знак долара прибрати і коми